src/scheduler/tasks/SynchronizeUsersImportTask.ts

```diff
@@ -19,12 +19,12 @@ export default class SynchronizeUsersImportTask extends SchedulerTask {
     const synchronizeUsersImport = LockingManager.createExclusiveLock(tenant.id, LockEntity.USER, 'synchronize-users-import');
```

Comment:
Same remarks as the Tag import, no need to put back same comments

Reply:
Done